mentat/session.py

```diff
@@ -78,6 +79,8 @@ async def _main(self):
             code_context.display_context()
             await conversation.display_token_count()
         except MentatError as e:
+            # BUG: this is not getting streamed in the Python and Terminal clients.
+            # It makes it look like Mentat is hanging forever.
```

Comment:
This might just happen when using Pytest. Need to verify.

Reply:
This looks like you haven't merged into main for a while; I think I fixed all the error stuff sometime last week.